.SRCINFO

```diff
@@ -1,7 +1,7 @@
 pkgbase = gotify-server
```

Comment:
I dunno, if we need migration script. Tho, I think if someone is using this package, it would be fair to include one.

What do you mean with `socket still missing`?

Reply:
since I'm moving directories back migration no longer required.

I mean systemd socket for socket-activation. So you don't need to run the service as long as it is not used.

I'm unhappy with the default port I chose though. 8080 is too common. Any suggestion?

Reply:
I've no preference. https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers maybe something like 2020?

Reply:
It's probably better to chose something that is not assigned here:
https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xml

(or open `/etc/service` which is part of `iana-etc` package)

Since this list is used by several applications to map names to port numbers (e.g `ss`), it would be very confusing to see `xinupageserver`.

Reply:
sure... just delegate this "easy" task to me :sweat_drops: 

I'll have a look at it

Reply:
xD. 4734, not listed in my /etc/services.

Reply:
Now that you offered...
**9071** (GOTI)

Fine with you?

Reply:
Yeah, great :D.